.github/workflows/close-stale-issues.yml

```diff
@@ -1,22 +1,22 @@
-name: "Close stale issues"
```

Comment:
There isn’t any r[elevant change in this file](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/6337/files#diff-221ec56332617dc2f33423aaada2c915da7a8a49d88ad5189606cfa78518fea9) for the tests right?

Reply:
Nope this is just linting corections